### DISCUSSION TITLE
Create a dedicated Rasperri Pi distro

### DISCUSSION THREAD

**Discussion by bufalo1973** [mastodon/mastodon#27926] [General]
### Pitch

To help distribute the load of the fediverse I think it would be a good idea to create a dedicated PC and RPi distro that needed little tweeking to set it and connect it. That way, anyone with an "old" (but not that old) PC or a RPi could help.

**Comment by grimneko** [comment]
A full blown distro would be a bit of an overkill, but maybe if you look around you can find something in the manner of a ansible playbook or similar that simplify the process. So it would be reduced to "install distro in basic mode" and execute playbook.

**Comment by bufalo1973** [comment]
I think the distro should be not only a Mastodon dedicated one but a full Fediverse distro, with the choice of running as many different aspects as wanted (Mastodon, PeerTube, Lemmy, KBin, ...), everything configurable with a web interface. And without anything that is not strictly necessary (no GCC, no Wayland, no X11, ...).

**Reply by yelvington** [reply]
https://github.com/YunoHost

https://github.com/YunoHost/apps

https://apps.yunohost.org/app/mastodon
